#158845195 Prompt user on new round

otseobande opened the pull request:
#### What does this PR do?

- Refactor app files
- Remove passport setup
- Add modal to prompt the user on a new round

#### Description of Task to be completed?
- N/A
#### How should this be manually tested?
- N/A
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
- #158845195
#### Screenshots (if appropriate)
![alert-user](https://user-images.githubusercontent.com/17123317/42654581-fefa10a2-8610-11e8-880c-a8e28bdecb01.gif)

#### Questions:
- N/A

iverenshaguy reviewed (approved):
Good work with the date conversion, test setup, and syntax fixes. Great job.

Stylll reviewed (approved):
Great job with this feature and also fixing up the db migration process. You have a keen eye for spotting mistakes and fixing bugs. Keep it up.

ozimos reviewed (approved):
Great Job, your refactoring of test setup ensures the team uses uniform DB cleanup methods after tests. Keep it up

otseobande committed:
feat(prompt): Prompt user on new round

- Refactor app files
- Remove passport setup
- Add modal to prompt user on new round

[Finishes #158845195]
